pkg/beholder/config.go

```diff
@@ -19,20 +19,45 @@ type Config struct {
 	EmitterExportTimeout time.Duration
 	// Batch processing is enabled by default
 	// Disable it only for testing
-	EmitterBatchProcessor bool
+	EmitterBatchProcessor      bool
+	EmitterExporterRetryConfig *RetryConfig
```

Comment:
Is this truly specific to the Emitter? If yes, should it be? i.e. why not `LogExporterRetryConfig`? :thinking:

Reply:
I plan to setup separate Exporter for Logger once we add support for it. Currently Emitter and Logger share otel sdk logger exporter

Reply:
Why is it a problem to share one logger exporter? It would be much simpler to have a single one registered globally.

Reply:
There could be noisy neighbor problem in case of verbose Logger. That can result in filling up internal exporter queue and dropped logs&custom-messages records. This is in theory, I have not verified that, and the benefit of having separate log exporter is also hypothetical as of now.

Reply:
But then this seems like an inconsistent starting point because we already have the global logger. Can we start with a global logger retry config and then introduce this field later if/when we separate them?

Reply:
Addressed here https://github.com/smartcontractkit/chainlink-common/pull/867/commits/da1ff9e8eedb326795ceb8493b3d5e2391e9f42a